src/OrchardCore.Modules/OrchardCore.Title/Handlers/TitlePartHandler.cs

```diff
@@ -64,8 +64,8 @@ private async Task SetTitleAsync(TitlePart part)
 
             var settings = GetSettings(part);
 
-            // Do not compute the title if the user can modify it.
-            if (settings.Options == TitlePartOptions.Editable || settings.Options == TitlePartOptions.EditableRequired)
+            // Do not compute the title if the user can modify it and the text is already set.
+            if ((settings.Options == TitlePartOptions.Editable || settings.Options == TitlePartOptions.EditableRequired)  && !String.IsNullOrWhiteSpace(part.ContentItem.DisplayText))
```

Comment:
What you are suggesting looks like we had before #12712.

But this is now contradictory with the below condition line 70 (also added recently by #12712) which would never be true, and then line 74 would never be executed.